### ISSUE TITLE
https credentials are not saved on the first attempt

### ISSUE THREAD

**Issue by cyberb** [maks/MGit#412]
1. Clone repo with https url
2. Make a commit
3. Push
4. Missing credentials sfreen (good)
5. Error send email message above it (bad)

You will need another set of steps (2 - 5) to actually save credentials.

**Comment by aisbergde** [comment]
yes, I need several attempts to get credential stored: when cloning, when pushing. I have no idea, where and how the credentials are stored and why they are not saved on first connection to a repository.

**Comment by cyberb** [comment]
Had to switch to termux + git + turboeditor (use storage access framework) long time ago.
But turboeditor sometimes has it's own issues saving to termux fs.
